### ISSUE TITLE
does not work on latest vscode v 1.91.1

### ISSUE THREAD

**Issue by themighty1** [itsyaasir/rust-feature-toggle#5]
The extension does not seem to work on vscode 1.91.1
I installed it from File-Preferences-Extensions and it shows as installed
There is no Status Bar menu for Rust Feature Toggler
Command Pallette (ctrl+shift+p) does show `Rust Feature Toggler: Toggle Feature` but selecting it gives an error popup: 
[
![Screenshot from 2024-07-16 11-22-38](https://github.com/user-attachments/assets/86301026-e8d8-401f-ac39-f66359164878)
](url)

**Comment by itsyaasir** [comment]
Hello @themighty1 sorry you are having this issue, I am currently using vscode 1.91.1 and I am not able to replicate this issue. I will do further research on it and see what is causing this. 

In the meantime, can you try removing/reinstalling the extension, reloading vscode and see if that fixes your issue. 

Thanks

**Comment by themighty1** [comment]
Thank you for confirming, I tried on a clean profile and your extension worked as expected. I guess one of the numerous extensions on my main profile is intervening. 
Debugging what exactly is happening would be more laborious than just manually typing up the features to enable in rust-analyzer's config, so I'll close this for now.

**Comment by icaroferre** [comment]
Just wanted to report that I have come across this issue on VS Code 1.99.3 and I have found a possible cause / solution.
It seems that the issue (at least in my case) was caused by a missing NPM package (toml). 
I'm not entirely sure what could have caused this issue since the extension was working fine last time I had to use it (perhaps it was caused by me installing / uninstalling some other extensions?).

Here's what I got from exthost.log:

```
2025-04-29 23:15:13.677 [info] ExtensionService#_doActivateExtension itsyaasir.rust-feature-toggler, startup: true, activationEvent: 'workspaceContains:Cargo.toml'
2025-04-29 23:15:13.679 [error] Activating extension itsyaasir.rust-feature-toggler failed due to an error:
2025-04-29 23:15:13.679 [error] Error: Cannot find module 'toml'
```

I was able to fix it by running ```cd ~/.vscode/extensions/itsyaasir.rust-feature-toggler-0.7.0``` and ```npm install toml```

Thank you for your work @itsyaasir! It's an incredibly helpful extension for Rust dev :)

**Comment by itsyaasir** [comment]
@icaroferre thanks for this! I was able to reproduce this and this has been fixed in v0.8.0; please install this and you won't require to manually install the node module
